### PR TITLE
Feature/always pp manual searched files

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

This PR will make sure that snatches done through a manual search, are always considered "priority" download. Meaning that although they might have already been downloaded in the past. A manual search result will always be post-processed.

### CODE REVIEW

**Comment by p0psicles** [comment]
@medariox there are comparable checks in process_tv.py and post_processor.py. It seems i need to make changes to both?

**Comment by p0psicles** [comment]
OK, so process_tv.py is the manual postprocessor which scans a folder. Then this looks like it could be a lot more DRY.

File: medusa/process_tv.py [hunk 1]
```diff
@@ -489,6 +489,29 @@ def unrar(self, path, rar_files, force=False):
...
+
+        if all(
+            [history_result,
+             history_result[0]['manually_searched'] == '1',
+             history_result[0]['action'] in (SNATCHED, SNATCHED_BEST, SNATCHED_PROPER)]
+        ):
+            self.log('Last snatch for this video was manual searched: {0}'.format(
+                video_file
+            ), logger.DEBUG)
+            return True
```
Comment (medariox) [inline medusa/process_tv.py]: The problem with that is that the `video_file` name doesn't always match the torrent/nzb name. So it's not 100% reliable.

File: medusa/post_processor.py [hunk 2]
```diff
@@ -1040,7 +1041,7 @@ def process(self):
                    (common.Quality.qualityStrings[new_ep_quality]), logger.DEBUG)
 
         # see if this is a priority download (is it snatched, in history, PROPER, or BEST)
-        priority_download = self._is_priority(old_ep_quality, new_ep_quality)
+        priority_download = self._is_priority(old_ep_quality, new_ep_quality) or self.manually_searched
```
Comment (medariox) [inline medusa/post_processor.py]: This will cause the file to be processed over and over if it's not moved away from the pp folder.

Comment (p0psicles) [reply medusa/post_processor.py]: Ok so your suggesting that we need to keep track of an additional flag. Which is enabled when it has been processed once?

Comment (medariox) [reply medusa/post_processor.py]: Sorry, I was actually wrong, it should work just fine in `post_processor` now. The issue was rather in `process_tv`. Should be good now.

**Comment by medariox** [comment]
@p0psicles Can you please let me know if it works for you this way? I've tested with a few releases and it seems to work well so far.

**Pull request merged**